variables.tf

```diff
@@ -266,6 +281,16 @@ variable "repositories" {
       teams       = ["distribution"]
     }
 
+    mageos-magento2-functional-testing-framework = {
```

Comment:
Just for sake of conformity, could you sort them alphabetically into the existing repositories? :see_no_evil: